documentation/working-with-testcafe-studio/recording-tests/test-actions/README.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: docs
+title: Test Actions
+permalink: /documentation/working-with-testcafe-studio/recording-tests/test-actions/
+---
+# Test Actions
+
+Topics of this section contain information about test actions that you can use in tests.
+These actions are available in the **Test Editor**'s **Actions** panel and organized into the following categories:
+
+Category              | Description
+--------------------- | --------------
+[Assertions](assertions.md) | Contains actions that perform different verifications.
+[On-Page Actions](on-page-actions.md) | Contains actions that simulate user actions on a tested page like clicks, key presses, drags, etc.
+[Debug](debug.md) | Contains actions for [debugging tests](../../debugging-tests.md).
+[Browser Actions](browser-actions.md) | Contains actions that perform various actions with a browser window.
+[Switch Frames](switch-frames.md) | Contains actions that allows you to switch the test's browsing context to iframes or back to the main window.
+
+<!--Each action has parameters that define how the action should be performed. When you add an action to the test, you need to specify its parameters. If you record a test, TestCafe Studio automatically sets up necessary parameters for the major of [on-page actions](on-page-actions.md). -->
+
+<!--You can add these actions to a test during or after recording. Some of them are added automatically during recording (see [On-Page Actions](#on-page-actions)).
+
+To add an action to a test manually, click its icons in the Actions panel. The action is added to the test. Then specify its parameters in the parameters area. If you add an action during recording, you can visually select a target element on the tested page, thus TestCafe Studio automatically specify the **Element Selector** parameter. See [During Recording](recording-tests/#during-recording). !!!!ADD TO THE Test Editor TOPIC-->
```

Comment:
I'd prefer to save this somewhere locally instead of commenting it out in the repository.